apps/web/src/data/subgraph/requests/exploreQueries.ts

```diff
@@ -46,12 +46,27 @@ export const exploreDaosRequest = async (
       settled: false,
     }
 
+    const first = 30
+
+    // filter spam daos from L2
+    if (
+      chainId === CHAIN_ID.BASE ||
+      chainId === CHAIN_ID.ZORA ||
+      chainId === CHAIN_ID.OPTIMISM
+    ) {
+      const activeDaos = await SDK.connect(chainId).activeDaos({
+        first,
+      })
+
+      // If we have less than 30 active daos, we apply the filter
+      if (activeDaos.daos.length !== first)
```

Comment:
I wanted to tie this to the page length. ie if we change page length of the explore page it will change this check as well. will change the comment to explain this better